tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -714,32 +713,7 @@ func postSchedulerConfigCommandFunc(cmd *cobra.Command, schedulerName string, ar
 		val = value
 	}
 	if schedulerName == "balance-hot-region-scheduler" && (key == "read-priorities" || key == "write-leader-priorities" || key == "write-peer-priorities") {
-		priorities := make([]string, 0)
-		prioritiesMap := make(map[string]struct{})
-		for _, priority := range strings.Split(value, ",") {
-			if priority != utils.BytePriority && priority != utils.KeyPriority && priority != utils.QueryPriority {
-				cmd.Println(fmt.Sprintf("priority should be one of [%s, %s, %s]",
-					utils.BytePriority,
-					utils.QueryPriority,
-					utils.KeyPriority))
-				return
-			}
-			if priority == utils.QueryPriority && key == "write-peer-priorities" {
-				cmd.Println("query is not allowed to be set in priorities for write-peer-priorities")
-				return
-			}
-			priorities = append(priorities, priority)
-			prioritiesMap[priority] = struct{}{}
-		}
-		if len(priorities) < 2 {
-			cmd.Println("priorities should have at least 2 dimensions")
-			return
-		}
-		input[key] = priorities
-		if len(priorities) != len(prioritiesMap) {
-			cmd.Println("priorities shouldn't be repeated")
-			return
-		}
+		input[key] = strings.Split(value, ",")
```

Comment:
why remove it? I think the check `priority should be one of [%s, %s, %s]` is necessary.

Reply:
http interface will check it also

Reply:
Can you point out the exact address which checked?

Reply:
https://github.com/tikv/pd/blob/39cff3bb2e5bea126212d92c90687ca63664d732/pkg/schedule/schedulers/hot_region_config.go#L352